corfudb-tools/src/main/java/org/corfudb/browser/CorfuStoreBrowser.java

```diff
@@ -198,18 +198,26 @@ public int listTables(String namespace)
         return numTables;
     }
 
+    private List<TableName> listTablesInNamespace(String namespace) {
+        return dynamicProtobufSerializer.getCachedRegistryTable().keySet()
+            .stream()
+            .filter(tableName -> namespace == null || tableName.getNamespace().equals(namespace))
+            .collect(Collectors.toList());
+    }
```

Comment:
Why do you use the cached registry and not the actual registry ? 

This is not robust, for example, imagine the logic of the DynamicProtobufSerializer is changed such that it does lazy loading, then this will return an incorrect view of the tables.

Reply:
DynamicProtobufSerializer is required for read/write operations to any tables.  

Before my change, some operations in the browser tool used ProtobufSerializer(actual registry) while others used the DynamicProtobufSerializer, making it difficult to understand and maintain the code.  To make it uniform for all browser operations, I have changed them all to use the DynamicaProtobufSerializer(and hence the cached registry).